2-collectors/scala-stream-collector/examples/config.hocon.sample

```diff
@@ -76,6 +76,19 @@ collector {
     placeholder = "[TOKEN]"
   }
 
+  # Optional, customize response handling for requests for the root path ("/").
+  # Useful if you need to redirect to web content or privacy policies regarding the use of this collector.
+  rootResponse {
+    # Optional, defaults to 404
+    statusCode = 302
+    headers = {
+        Location = "https://127.0.0.1/",
+        X-Custom = "something"
+    }
+    # Optional, defaults to empty string
+    body = "302, redirecting"
```

Comment:
Do we need a content type?

Reply:
I was curious if the entire HttpEntity could be stored in HOCON, but didn't push it that far.

Reply:
can't the content-type be specified through the headers anyway?